tests/by_image/base-notebook/test_healthcheck.py

```diff
@@ -11,6 +11,32 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def get_healthy_status(
+    container: TrackedContainer,
+    env: list[str] | None,
+    cmd: list[str] | None,
+    user: str | None,
+) -> str:
+    running_container = container.run_detached(
+        tty=True,
+        environment=env,
+        command=cmd,
+        user=user,
+    )
+
+    # giving some time to let the server start
+    finish_time = time.time() + 10
+    sleep_time = 1
```

Comment:
Sleeping 1 second instead of 0.1, because otherwise we produce too much logs when running `pytest` directly